vtkext/private/module/vtkF3DRenderer.cxx

```diff
@@ -1551,9 +1515,7 @@ void vtkF3DRenderer::Render()
   fps = std::min(fps, static_cast<int>(std::round(1.0 / (elapsed * 1e-9))));
 #endif
 
-  std::string str = std::to_string(fps);
-  str += " fps";
-  this->TimerActor->SetInput(str.c_str());
+  this->UIActor->SetFpsValue(fps);
```

Comment:
is the "double" render still needed to get correct fps ? see interactor_impl.cxx

Reply:
Can you point out the line number please?

Reply:
https://github.com/f3d-app/f3d/blob/master/library/src/interactor_impl.cxx#L627

this line may be removable, maybe

Reply:
indeed! I removed the special command